src/server/browserContext.ts

```diff
@@ -299,6 +303,26 @@ export abstract class BrowserContext extends SdkObject {
     await this._closePromise;
   }
 
+  requestStarted(request: network.Request) {
```

Comment:
Sorry for missing it earlier, but instead of these methods, let's emit browser context  events from the FrameManager, in addition to page events. That should be easier.

Reply:
Done